addons/account/models/chart_template.py

```diff
@@ -264,6 +264,8 @@ def _pre_reload_data(self, company, template_data, data, force_create=True):
             if prop.startswith('property_'):
                 template_data.pop(prop)
         data.pop('account.reconcile.model', None)
+        if not force_create:
+            data.pop('res.company', None)
```

Comment:
The issues here are the same as for the line below with `anglo_saxon_accounting`; but they are not related to `force_create`.
We never want to update such fields: whether during the upgrade or when synchronizing the CoA to fetch newer data.